### PR TITLE
bare-metal-beta-installer: Includes disk encryption option.

### PR DESCRIPTION
**Pull request by mvincerx**
Adds new Figures 11, 12, 13, and 19;

Revise filenames and Figures succeeding Fig 11 accordingly.

Signed-off-by: Michael Vincerra <michael.vincerra@intel.com>

### CODE REVIEW

**Comment by mvincerx** [comment]
@mdhorn, Please review and provide feedback/approval at your earliest convenience.

**Comment by mvincerx** [comment]
@ulissesf FYI only. Adding new root encryption for live-desktop-beta-installer.

File: source/clear-linux/get-started/bare-metal-install-beta/bare-metal-install-beta.rst [hunk 1]
```diff
@@ -307,6 +307,28 @@ requirements shown in Table 1:
      - /
      - *Size depends upon use case/desired bundles.*
 
+Disk encryption
+---------------
+
+Disk encryption is supported for root and swap partitions on |CL| for
```
Comment (mdhorn) [inline source/clear-linux/get-started/bare-metal-install-beta/bare-metal-install-beta.rst]: Disk encryption is supported by use of LUKS for all partitions with the exception of 'boot' on |CL| for

File: source/clear-linux/get-started/bare-metal-install-beta/bare-metal-install-beta.rst [hunk 2]
```diff
@@ -307,6 +307,28 @@ requirements shown in Table 1:
...
+.. note::
+
+   **The `boot` partition cannot be encrypted**.
+
+Encryption Passphrase
+---------------------
+
+|CL| supports use of a single passphrase for all encrypted partitions.
+Therefore, if you turn on encryption for one partition, the same passphrase
+must be used for all partitions. For example, suppose you encrypt the `root`
```
Comment (mdhorn) [inline source/clear-linux/get-started/bare-metal-install-beta/bare-metal-install-beta.rst]: will be used for all partitions. For example, suppose you encrypt the `root`

File: source/clear-linux/get-started/bare-metal-install-beta/bare-metal-install-beta.rst [hunk 3]
```diff
@@ -307,6 +307,28 @@ requirements shown in Table 1:
...
+partition first and the `swap` partition second. Upon selecting encryption
+for the `swap` partition, the :guilabel:`Encryption Passphrase` dialogue
+appears showing the current default Passphrase. Select :guilabel:`Confirm`
+and press :kbd:`Enter` to use the same Passphrase.
+
+
```
Comment (mdhorn) [inline source/clear-linux/get-started/bare-metal-install-beta/bare-metal-install-beta.rst]: .. note::

    **Additional or different keys may be configured post-installation using `cryptsetup` tool.**.

**Comment by tpleavitt** [comment]
@mdhorn you can do a git clone of the clear-linux-documentation repo, cd into clear-linux-documentation and then:
make html

view new output in build/html/html.index

**Comment by tpleavitt** [comment]
@mdhorn meant to say pull the branch with the changes you want to see and then do the make

File: source/clear-linux/get-started/bare-metal-install-beta/bare-metal-install-beta.rst (hunk 1 above)
Comment (mvincerx) [reply source/clear-linux/get-started/bare-metal-install-beta/bare-metal-install-beta.rst]: @mdhorn. OK. I will refine the description per your response.  Also, to view the images in-line, select the "View" button in the upper right. BTW: I will change Fig. 11; it needs to show "Encrpyt" checked.
Here's the direct link to view images in line.
https://github.com/clearlinux/clear-linux-documentation/blob/da0e559c04b804abc2295fc2b69966777cbd0bc2/source/clear-linux/get-started/bare-metal-install-beta/bare-metal-install-beta.rst#advanced-options

**Comment by mvincerx** [comment]
@tpleavitt . Thanks. I can help @mdhorn today. We're meeting at 13:00 in the office.

**Comment by mvincerx** [comment]
@mdhorn. Please review my latest commit and let me know.

File: source/clear-linux/get-started/bare-metal-install-beta/bare-metal-install-beta.rst [hunk 4]
```diff
@@ -279,13 +279,35 @@ Auto partition will configure the media to meet the minimum requirements.
...
+===============
+
+For greater security, disk encryption is supported by use of LUKS for all
+partitions **except boot** on |CL|. Encryption is optional. To encrypt a
+partition, see the example below to encrypt :ref:`encrypt-root`.
+
+Encryption Passphrase
+---------------------
+
+|CL| supports use of a single passphrase for all encrypted partitions.
```
Comment (mdhorn) [inline source/clear-linux/get-started/bare-metal-install-beta/bare-metal-install-beta.rst]: |CL| uses a single passphrase for all encrypted partitions.

File: source/clear-linux/get-started/bare-metal-install-beta/bare-metal-install-beta.rst [hunk 5]
```diff
@@ -279,13 +279,35 @@ Auto partition will configure the media to meet the minimum requirements.
...
+is used for all partitions.
+
+.. note::
+
+   For example, suppose you encrypt the `root` partition first and the `swap`
+   partition second. Upon selecting encryption for the `swap` partition, the
+   :guilabel:`Encryption Passphrase` dialogue appears showing the default
+   Passphrase assigned to the `root`. Select :guilabel:`Confirm` and press
+   :kbd:`Enter` to use the same Passphrase.
+
```
Comment (mdhorn) [inline source/clear-linux/get-started/bare-metal-install-beta/bare-metal-install-beta.rst]: **Additional or different keys may be configured post-installation using `cryptsetup` tool.**

File: source/clear-linux/get-started/bare-metal-install-beta/bare-metal-install-beta.rst [hunk 6]
```diff
@@ -752,11 +802,11 @@ Assign Hostname
       alphanumeric character but may also contain hyphens. Maximum length of
       63 characters.
 
-   .. figure:: figures/bare-metal-install-beta-26.png
+   .. figure:: figures/bare-metal-install-beta-27.png
```
Comment (mdhorn) [inline source/clear-linux/get-started/bare-metal-install-beta/bare-metal-install-beta.rst]: For some reason, the hostname screenshot is appearing much larger than all other screenshots when I use preview viewing.

File: source/clear-linux/get-started/bare-metal-install-beta/bare-metal-install-beta.rst (hunk 3 above)
Comment (mvincerx) [reply source/clear-linux/get-started/bare-metal-install-beta/bare-metal-install-beta.rst]: Resolved above. See next commit.

File: source/clear-linux/get-started/bare-metal-install-beta/bare-metal-install-beta.rst (hunk 6 above)
Comment (mvincerx) [reply source/clear-linux/get-started/bare-metal-install-beta/bare-metal-install-beta.rst]: Revised. Thanks.

**Comment by mvincerx** [comment]
@mdhorn Please review at your earliest convenience. I'd like to merge this before I start next PR--as discussed.

**Pull request merged**